Dockerfile

```diff
@@ -1,3 +1,43 @@
+##
+# Build Stage
+#
+FROM ubuntu:focal as build
+
+ENV DEFAULT_PW="adminADMIN!"
```

Comment:
@pmanko , since this variable is required at  at build time of the image , 
i think  we better pass it as `ARG` , instead of 'ENV' ,so that it can easily be passed to the builder action at build time